### PR TITLE
Add a summarization script for internal fragmentation analysis

### PR DESCRIPTION
**Pull request by yinan1048576**

### CODE REVIEW

**Comment by yinan1048576** [comment]
Modify column names to match changes in https://github.com/jemalloc/jemalloc/pull/1914.

File: summarize-internal-frag.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# See summarize.py for the requirement on the input; the only exception is that
+# this script processes the table format of malloc stats because the json
+# format does not have the necessary information as of the time this script was
```
Comment (davidtgoldblatt) [inline summarize-internal-frag.py]: I mean, we do now, right? We should just parse these out of json, rather than having a brittle manual parsing setup.

**Comment by yinan1048576** [comment]
Rewrote to parse the JSON output instead.

**Review by davidtgoldblatt — APPROVED** [review]

**Comment by yinan1048576** [comment]
FYI I don't think I have the permission to merge.

**Pull request merged**